charts/backstage/values.yaml

```diff
@@ -121,6 +121,15 @@ backstage:
   extraVolumeMounts: []
   extraVolumes: []
 
+  # Allows to define the appConfig as a multiline string that generates a ConfigMap
+  # automatically, not requiring to have it pre provisioned as with the extraAppConfig key.
+  # DO NOT USE if you need to put sensitive data in the appConfig.
+  # E.g:
+  # appConfig: |
+  #   app:
+  #     baseUrl: https://somedomain.tld
+  appConfig: null
```

Comment:
Is there any reason why to store the config as a string? Backstage configs are defined as a dictionary. We can do the same here, wdyt?

Reply:
Well, I think that would be the best, configuring Backstage directly from the keys on the `values.yaml`. That should add up to generate the ConfigMap directly or environment variables (if possible). But I do not really know all the Backstage configurations.

Also to mention, that would make this chart dependent on new Backstage configurations.

Reply:
Which is the difference with the current code?

```yaml
app-config.yaml: {{ toYaml .Values.backstage.appConfig | indent 2 }}
```

toYaml should verify that it is a valid yaml code. Is it just to see from the IDE the config as yaml instead of a multiline string?

About the default value, sure, if we switch it to be a dictionary.

Reply:
Got it and agree :) Updated now